gravitee-apim-portal-webui-next/src/app/applications/create-application/create-application.component.html

```diff
@@ -20,5 +20,163 @@
   <mat-card-content>
     <h2 i18n="@@applicationsCreateTitle">Create new application</h2>
     <p i18n="@@applicationsCreateDescription">An application is used to register and agree to plans.</p>
+
+    @if (enabledApplicationTypes(); as types) {
+      <div class="cards-container" [appIsMobile]="'cards-container--mobile'">
+        <mat-card appearance="outlined">
+          <mat-card-content>
+            <h3 class="section-title" i18n="@@createApplicationSecuritySetupTitle">Set up security</h3>
+            @if (isMobile()) {
+              <mat-form-field appearance="outline" class="type-select-mobile">
+                <mat-label i18n="@@createApplicationTypeTitleMobile">Type</mat-label>
+                <mat-select [formControl]="typeIdControl">
+                  @for (type of types; track type.id ?? type.name) {
```

Comment:
If you want to avoid the check, you can choose one of the internal values:
```suggestion
                  @for (type of types; track $index) {
```

Reply:
sure, good call, thanks